lib/irclib/client.js

```diff
@@ -13,6 +13,12 @@ var log = require("../logging").get("irc-client");
 // The length of time to wait before trying to join the channel again
 var JOIN_TIMEOUT_MS = 15 * 1000; // 15s
 
+// A global counter for generating unique numerical user IDs
+var userCount = 0;
```

Comment:
This will not create a unique 1:1 mapping from user ID to IRC client. Restarting the bridge will lead to users getting each others IPv6 addresses. Given these addresses are used for identification for bans etc, we can't really do this. They need to be calculated and stored similar to ident usernames.